tarogarden/planter.go

```diff
@@ -47,7 +47,7 @@ type PlanterConfig struct {
 
 	// BatchTicker is used to notify the planter than it should assemble
 	// all asset requests into a new batch.
-	BatchTicker ticker.Ticker
+	BatchTicker *ticker.Force
```

Comment:
> LGTM, I guess that we will add an API/cli way to force the batch later?

Yeah we can, I figured wasn't a priority here since I'd want to make other CLI changes first.